plugins/main/host-device/host-device_test.go

```diff
@@ -234,6 +234,7 @@ var _ = Describe("base functionality", func() {
 
 	AfterEach(func() {
 		originalNS.Close()
```

Comment:
Could you help to change this to `Expect(originalNS.Close()).To(Succeed())` ?

Reply:
Yes, I will do it, thank you @mars1024